python/pipz/version.py

```diff
@@ -1,6 +1,6 @@
 
 try:
-    from __version__ import version
+    from .__version__ import version
```

Comment:
How strange, I've been using this almost exclusively with Python 3 so far. That this never errored I cannot grasp. :O